glide.yaml

```diff
@@ -21,3 +21,4 @@ testImport:
   vcs: git
   subpackages:
   - golint
+- package: go.uber.org/goleak
```

Comment:
this is only in `testImport`, so it shouldn't cause any issues for other packages that depend on fx

Reply:
sure. however, dep and vgo have no concepts of test imports.
so either way this is only a can that gets kicked so far down the road

Reply:
They don't have the concept, but they traverse the dependency graph, so as long as it's only ever used in `_test.go` files, dependents of this package won't try to take on `goleak`.

Separately, it doesn't hurt to cut a release of goleak, I just haven't spent the time vetting the interface we should 1.0 with.

Reply:
added a version `^0.10`

Reply:
Please change to `~0.10`. In pre-1.0 packages, we should lock to a minor version range instead of a major.

Reply:
for glide/dep, carrot on a zero major actually means minor lock :/

`^0.10` is effectively `~0.10` as I recently discovered.

That said I'm more in favor of `~0.10` rather than having to explain the `^0` every time. My point was that it's functionally the same.

Reply:
https://golang.github.io/dep/docs/Gopkg.toml.html#version 

/shrug

Reply:
Isn't that only true of dep, and not glide?